From ce304b131e12216046f5d3c9ba4e3f1e582be11d Mon Sep 17 00:00:00 2001
From: Charles Stevens <charles.stevens@compasses.com>
Date: Mon, 9 Nov 2020 10:53:44 -0600
Subject: [PATCH 31/44] RSOM-186

Quiet warning message related to unsupport BCLK divisor. This message is only
relevent when runing in I2S master mode.

Signed-off-by: Charles Stevens <charles.stevens@compasses.com>
---
 sound/soc/codecs/wm8962.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/wm8962.c b/sound/soc/codecs/wm8962.c
index dde015fd70a4..779ee957e176 100644
--- a/sound/soc/codecs/wm8962.c
+++ b/sound/soc/codecs/wm8962.c
@@ -2429,7 +2429,7 @@ static void wm8962_configure_bclk(struct snd_soc_component *component)
 	}
 
 	if (i == ARRAY_SIZE(sysclk_rates)) {
-		dev_err(component->dev, "Unsupported sysclk ratio %d\n",
+		dev_dbg(component->dev, "Unsupported sysclk ratio %d\n",
 			wm8962->sysclk_rate / wm8962->lrclk);
 		return;
 	}
-- 
2.17.1

